### DISCUSSION TITLE
noticed a few things with this slicer and profiles

### DISCUSSION THREAD

**Discussion by Edzi11** [OrcaSlicer/OrcaSlicer#1115] [General]
So ive been having issues with my supports lately in terms of how the 1.6.2 version generates them in mid-air. I decided to give the new beta version a try and was having some issues getting the network plugin installed. tried a lot to get it to install and it wasn't until i closed out all orca instances that it finally installed the plugin. to sum it up it appears that you can only have one version of the slicer open at a time at least for me. when i re-opened 1.6.2 it installed the plugin again and worked. When i opened up the new beta it had to re-install the plugin again as well. very odd that it does that with orca but it doesnt seem to be an issue with the official bambu slicer as i can have both orca and bambu open at the same time.

Another thing which I'm not sure is an issue or not but when on the new beta it didn't transfer any of my filament or user presets. My printer profiles were there but not the other two profiles and when trying to import the profiles over from the previous slicer it didnt want to import them which im guessing is maybe theyre not compatible with this new beta. 

these are just rambling but im curious is any of yall have run into stuff like this.

**Comment by fr3ak2402** [comment]
@Edzi11 

For your profiles you have to adjust something to make them work in OrcaSlicer again.
This has already been implemented in this issue, just look at the issue. If you have questions etc, I will try to help you.
#1107

**Reply by Edzi11** [reply]
I did actually find that thread but my profiles and processes were under a different folder for some reason. My profiles and such just randomly appeared this morning and i didn't do anything so not sure what fixed it.
